app/components/ActivityColumn/ActivityColumn.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { ActivityCell } from '../ActivityCell';
+
+import CommitActivityWeek from '../../vos/CommitActivityWeek';
+
+import './ActivityColumn.scss';
+
+export class ActivityColumn extends Component {
+  static propTypes = {
+    weekData : PropTypes.instanceOf(CommitActivityWeek)
+  };
+
+  static defaultProps = {
+    weekData: new CommitActivityWeek()
+  };
+
+  commitDensity (commits) {
```

Comment:
Commit density should be a dynamic scale based on the lowest / highest commits for the given repo.

example: 
If the highest commit activity per day is 80 commits
then 
2 = 1-20 commits
3 = 21-40 commits
4 = 41-60 commits
5 = 61-80 commits